Camera2Extensions/app/src/main/java/com/example/android/camera2/extensions/fragments/CameraFragment.kt

```diff
@@ -790,9 +806,11 @@ class CameraFragment : Fragment(), TextureView.SurfaceTextureListener {
     if (!jpegSupportedSizes.isEmpty()) {
       postviewSize = jpegSupportedSizes[0]
       postviewFormat = ImageFormat.JPEG
-    } else {
+    } else if (!yuvSupportedSizes.isEmpty()){
```

Comment:
and same here: nit: let's change this to `if (yuvSupportedSizes.isNotEmpty()) {`

Reply:
Done